types/V1.mli

```diff
@@ -48,10 +48,23 @@ open Result
 
     {e Release %%VERSION%% } *)
 
-(** {1 Abstract devices}
+type 'a pp = Format.formatter -> 'a -> unit
```

Comment:
One thing that would help beginners (as a documentation pass) is some tips on where to find functions that use the defined types -- for example, a pointer to Ptime for the time functions, Fmt for the pretty printers, Rresult for result combinators.  This can be be done later once the code compiles :)